dynamic-storage.html.md.erb

```diff
@@ -179,3 +179,123 @@ To get more information about the underlying EMC ECS product, please consult the
 * GET Object torrent
 
 Using a S3 compatible URL, Dynamic Storage can easily be used as a backend.
+
+#### Best Practices
+
+#### App Design
+
+•	Don’t require low latency
+•	Don’t execute serial workflows.  Process many operations in parallel.
+•	Be prepared to handle failures
+	–	Most SDKs have built-in retry mechanisms to handle this for you
```

Comment:
Indent these by two spaces instead of a tab and also use asterisks as the bullet points.